TheraLang.BLL/Services/MemberFeeService.cs

```diff
@@ -59,6 +60,12 @@ public async Task AddAsync(MemberFeeDto memberFeeDto)
                         $"{nameof(MemberFeeDto)} cannot be null");
                 }
 
+                var minDate = GetMemberFeesAsync().Result.Last().FeeDate.AddMonths(1);
+                if (memberFeeDto.FeeDate <= minDate)
+                {
+                    throw new Exception(
```

Comment:
Ne, tak dela ne delayutsa.
You should use ApiException if you manually throw exception. Maybe you should create your exception that inherits from ApiException if there is no appropriate.

Reply:
Done)))